include/atomics/receiverCadmium.hpp

```diff
@@ -30,84 +29,86 @@ using namespace cadmium;
 using namespace std;
 
 //Port definition
-    struct Receiver_defs{
-        struct out : public out_port<Message_t> {
-        };
-        struct in : public in_port<Message_t> {
-        };
-    };
+// changed Receiver_defs to receiver_defs Syed Omar
+struct receiver_defs{
+    struct out : public out_port<message_t> {};
+    struct in : public in_port<message_t> {};
+};
    
-    template<typename TIME>
-    class Receiver{
-        using defs=Receiver_defs; // putting definitions in context
-        public:
-            //Parameters to be overwriten when instantiating the atomic model
-            TIME   preparationTime;
-            // default constructor
-            Receiver() noexcept{
-              preparationTime  = TIME("00:00:10");
-              state.ackNum    = 0;
-              state.sending     = false;
-            }
+template<typename TIME>
+class Receiver{
+    using defs = receiver_defs; // putting definitions in context
+    public:
+    //Parameters to be overwriten when instantiating the atomic model
+    TIME preparation_time;

```

Comment:
preparation_time variable can be considered a constant, so it can be in capital letters

Reply:
resolved the issue. Please review the latest code